specification/scvmm/ScVmm.Management/examples/2024-06-01/VirtualMachineInstances_CreateOrUpdate_MaximumSet_Gen.json

```diff
@@ -0,0 +1,329 @@
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "name": "kvofzqulbjlbtt",
+              "macAddress": "oaeqqegt",
+              "virtualNetworkId": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.ScVmm/virtualNetworks/virtualNetworkName",
+              "ipv4AddressType": "Dynamic",
+              "ipv6AddressType": "Dynamic",
+              "macAddressType": "Dynamic",
+              "nicId": "roxpsvlo"
+            }
+          ]
+        },
+        "storageProfile": {
+          "disks": [
+            {
+              "name": "fgnckfymwdsqnfxkdvexuaobe",
+              "diskId": "ltdrwcfjklpsimhzqyh",
+              "diskSizeGB": 30,
+              "bus": 8,
+              "lun": 10,
+              "busType": "zu",
+              "vhdType": "cnbeeeylrvopigdynvgpkfp",
+              "templateDiskId": "lcdwrokpyvekqccclf",
+              "storageQoSPolicy": {
+                "name": "ceiyfrflu",
+                "id": "o"
+              },
+              "createDiffDisk": "true"
+            }
+          ]
+        },
+        "infrastructureProfile": {
+          "inventoryItemId": "ihkkqmg",
+          "vmmServerId": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.ScVmm/vmmServers/vmmServerName",
+          "cloudId": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.ScVmm/clouds/cloudResourceName",
+          "templateId": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.ScVmm/virtualMachineTemplates/virtualMachineTemplateName",
+          "vmName": "qovpayfydhcvfrhe",
+          "uuid": "hrpw",
+          "lastRestoredVMCheckpoint": {
+            "parentCheckpointID": "hqhhzikoxunuqguouw",
+            "checkpointID": "wsqmrje",
+            "name": "keqn",
+            "description": "qurzfrgyflrh"
+          },
+          "checkpointType": "jkbpzjxpeegackhsvikrnlnwqz",
+          "generation": 28,
+          "biosGuid": "xixivxifyql"
+        }
+      },
+      "extendedLocation": {
+        "type": "customLocation",
+        "name": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.ExtendedLocation/customLocations/customLocationName"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "availabilitySets": [
+            {
+              "id": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.ScVmm/availabilitySets/availabilitySetResourceName",
+              "name": "lwbhaseo"
+            }
+          ],
+          "osProfile": {
+            "computerName": "uuxpcxuxcufllc",
+            "osType": "Windows",
+            "osSku": "cxqnjxgkts",
+            "osVersion": "djt"
```

Comment:
newly added osProfile properties are missing in this response.

Reply:
Update Get & List APIs as well with newly added properties.

Reply:
Done